src/test/scripts/download_sbml-test-suite.sh

```diff
@@ -6,14 +6,15 @@
 #  source ./download_sbml-test-suite.sh
 #################################################
 
+set -euo pipefail
+
 # This URL gives the updated SBML test cases from SBML Test Suite repository
 SBML_TEST_SUITE_LINK="https://github.com/sbmlteam/sbml-test-suite/branches/develop/cases"
 
 _CWD="$PWD"
 TEST_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 # download and extract the bigg models for testing
-cd $TEST_DIR/../resources/sbml-test-suite
```

Comment:
This looks like a copy-paste mistake: it only makes sense to `cd` after the `rm` and `mkdir` (see following lines).